### PR TITLE
Fixup/company v3 corrections

### PR DESCRIPTION
**Pull request by jim68000**

### CODE REVIEW

File: generator/specs/definitions/company_defs.yml [hunk 1]
```diff
@@ -119,7 +107,7 @@
         $ref: '#/definitions/TurnoverRange'
       uk_region:
         $ref: '#/definitions/UKRegion'
-      investment_projects:
+      investment_projects_invested_in:
```
Comment (reupen) [inline generator/specs/definitions/company_defs.yml]: Would prefer to break out the count to a separate field (`investment_projects_invested_in_count`) – for consistency with contacts and simpler implementation.

**Comment by reupen** [comment]
Can we add unarchive back in?

File: generator/specs/definitions/company_defs.yml (hunk 1 above)
Comment (jim68000) [inline generator/specs/definitions/company_defs.yml]: OK

**Comment by jim68000** [comment]
Sorry, yeah, forgot unarch

**Pull request merged**